src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
The class `Ball` should have an access modifier to define its visibility explicitly. The default package-private visibility may not be intended.